go.mod

```diff
@@ -5,7 +5,7 @@ go 1.16 // please change also in repo secrets
 require (
 	github.com/adhocore/gronx v0.2.6
 	github.com/go-resty/resty/v2 v2.6.0
-	github.com/golang/mock v1.4.3
+	github.com/golang/mock v1.6.0
```

Comment:
gomock had some issues with checking equality between maps.
This issue was fixed in newer versions (I couldn't get my tests to pass with 1.4.3 due to false inequality).